### DISCUSSION TITLE
umi4运行时配置layout报错

### DISCUSSION THREAD

**Discussion by qingfengdai** [umijs/umi#8350] [Q&A]
按umi max的文档试了一下修改公共layout,运行直接报错,不知道哪里配置不对
![9C3689D4-E293-408c-8693-D2F5F32AF2FA](https://user-images.githubusercontent.com/38605341/176982592-f5aaf715-c975-415e-a746-6d851d6a7340.png)

报错如下:
![416B5711-CEE3-4c82-9865-3E149ECB1509](https://user-images.githubusercontent.com/38605341/176982630-ee1c7a0a-014f-47ab-812a-62a69dc567a7.png)

**Comment by fz6m** [comment]
尝试重命名 `app.ts` to `app.tsx`

**Comment by qingfengdai** [comment]
好了,谢谢
